docs/src/markdownDocs/environment.mdx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Divider } from '../layout/Divider';
+import { CodeBlock } from '../shared';
+
+{/* KEYWORDS: developer develop environment guide tutorial */}
+
+# Setting up Your Development Environment
+
+Getting started is often the most frustrating part of application development. Fortunately, many of the modern JavaScript frameworks have made great strides in providing tools to get you up and running as quickly as possible. Below, we walk through the basics of getting your system set up to develop.
+
+## Integrated Development Environment (IDE)
+
+There are a bunch of different IDEs on the market, and in general you should be able to use any of these that you like and are comfortable with. If you don't have a personal preference, we recommend trying [Visual Studio Code](https://code.visualstudio.com) - it is quite excellent and comes with a variety of tools to make your development experience seamless. And it's free!
+
+## Git
+
+Chances are, you probably want to keep your application in source control. Git is one of the big players in this space (it's also the backbone for BitBucket).
+
+You may already have git installed. To check, run:
+
+<CodeBlock sx={{ height: 'auto', fontSize: '14px' }} code={`git --version`} />
```

Comment:
looks good, but is there a way to prevent you from having to pass this `sx={{ height: 'auto', fontSize: '14px' }}` all the time? Can you define the style in the component, and use a prop to toggle it on?